incubator/cockroachdb/README.md

```diff
@@ -0,0 +1,184 @@
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install --name my-release -f values.yaml incubator/cockroachdb
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+# Deep dive
+
+## Connecting to the CockroachDB cluster
+
+Once you've created the cluster, you can start talking to it it by connecting
+to its "public" service. CockroachDB is PostgreSQL wire protocol compatible so
+there's a [wide variety of supported
+clients](https://www.cockroachlabs.com/docs/install-client-drivers.html). For
+the sake of example, we'll open up a SQL shell using CockroachDB's built-in
+shell and play around with it a bit, like this (likely needing to replace
+"my-release-cockroachdb-public" with the name of the "-public" service that
+was created with your installed chart):
+
+```console
+$ kubectl run -it --rm cockroach-client \
+    --image=cockroachdb/cockroach \
+    --restart=Never \
+    --command -- ./cockroach sql --host my-release-cockroachdb-public
+Waiting for pod default/cockroach-client to be running, status is Pending,
+pod ready: false
+If you don't see a command prompt, try pressing enter.
+root@my-release-cockroachdb-public:26257> SHOW DATABASES;
++--------------------+
+|      Database      |
++--------------------+
+| information_schema |
+| pg_catalog         |
+| system             |
++--------------------+
+(3 rows)
+root@my-release-cockroachdb-public:26257> CREATE DATABASE bank;
+CREATE DATABASE
+root@my-release-cockroachdb-public:26257> CREATE TABLE bank.accounts (id INT
+PRIMARY KEY, balance DECIMAL);
+CREATE TABLE
+root@my-release-cockroachdb-public:26257> INSERT INTO bank.accounts VALUES
+(1234, 10000.50);
+INSERT 1
+root@my-release-cockroachdb-public:26257> SELECT * FROM bank.accounts;
++------+---------+
+|  id  | balance |
++------+---------+
+| 1234 | 10000.5 |
++------+---------+
+(1 row)
+root@my-release-cockroachdb-public:26257> \q
+Waiting for pod default/cockroach-client to terminate, status is Running
+pod "cockroach-client" deleted
+```
+
+## Cluster health
+
+Because our pod spec includes regular health checks of the CockroachDB processes,
+simply running `kubectl get pods` and looking at the `STATUS` column is sufficient
+to determine the health of each instance in the cluster.
+
+If you want more detailed information about the cluster, the best place to look
+is the admin UI.
+
+## Accessing the admin UI
+
+If you want to see information about how the cluster is doing, you can try
+pulling up the CockroachDB admin UI by port-forwarding from your local machine
+to one of the pods (replacing "release-cockroachdb-0" with one of your pods' names):
+
+```shell
+kubectl port-forward release-cockroachdb-0 8080
```

Comment:
No, that's a good point. I've fixed up the couple inconsistent ones, now using shell without a $ for anything where we don't care to show command output and console with $ for anything where we do.